### PR TITLE
DATAUP-338: Removing font awesome css inclusion II

### PR DESCRIPTION
**Pull request by ialarmedalien**
# Description of PR purpose/changes

Pt II of removing font-awesome css inclusion: deleting the inclusion statements from individual JS files.

I also cleaned up any places where I found the pattern `css!path/to/file.css` and replaced it with `css!path/to/file`.

Both sets of frontend unit tests passed!!! This must be a miracle!

# Jira Ticket / Issue #
e.g. https://kbase-jira.atlassian.net/browse/DATAUP-338
- [x] Added the Jira Ticket to the title of the PR (e.g. `DATAUP-69 Adds a PR template`)

# Testing Instructions
* Details for how to test the PR:
- [x] Tests pass locally and in GitHub Actions
- [] Changes available by spinning up a local narrative and navigating to _X_ to see _Y_

# Dev Checklist:

- [x] My code follows the guidelines at https://sites.google.com/lbl.gov/trussresources/home?authuser=0
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] (JavaScript) I have run Prettier and ESLint on changed code manually or with a git precommit hook

### CODE REVIEW

File: kbase-extension/static/kbase/js/widgets/appWidgets2/fieldWidgetMicro.js [hunk 1]
```diff
@@ -388,8 +388,7 @@ define([
                                     div(
                                         {
                                             id: ids.feedback,
-                                            class:
-                                                'input-group-addon kb-input-group-addon kb-app-field-feedback',
+                                            class: 'input-group-addon kb-input-group-addon kb-app-field-feedback',
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/js/widgets/appWidgets2/fieldWidgetMicro.js:391]: our non-idempotent file formatting pal

File: nbextensions/viewCell/widgets/viewCellWidget.js [hunk 2]
```diff
@@ -1,1343 +1,1355 @@
...
-                    elements: {
-                        show: [],
-                        hide: [
-                            'fatal-error',
-                            'parameters-group',
-                            'output-group',
-                            'parameters-display-group',
-                            'exec-group',
-                        ],
+                    ui: {
```
Comment (ialarmedalien) [inline nbextensions/viewCell/widgets/viewCellWidget.js:66]: line 65 is obviously different from line 64 in the old file. 🤦

File: nbextensions/editorCell/widgets/editors/readsSet/editor.js [hunk 3]
```diff
@@ -1,1376 +1,1401 @@
...
-        let fsm;
-
-        if (runtime.config('features.developer')) {
-            settings.showDeveloper = {
-                label: 'Show developer features',
-                defaultValue: false,
-                type: 'toggle',
-                element: 'developer-options',
-            };
+define(
```
[the remark this answers is not in the document]
Comment (briehl) [reply nbextensions/editorCell/widgets/editors/readsSet/editor.js:1]: Thanks, Prettier! 😵

File: nbextensions/viewCell/widgets/viewCellWidget.js (hunk 2 above)
Comment (briehl) [reply nbextensions/viewCell/widgets/viewCellWidget.js:66]: Prettier + Git = 🤯

Comment (briehl) [reply nbextensions/viewCell/widgets/viewCellWidget.js:66]: Just to verify, the only changes here were the CSS imports, right?

**Pull request merged**